unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Settings/Settings.cs

```diff
@@ -113,6 +113,8 @@ private SettingsData.GeneralSettings GetDefaultGeneralSettings()
                 sfxVolume = 1,
                 mouseSensitivity = 0.6f,
                 scenesLoadRadius = 4,
+                avatarsLODDistance = 16,
+                maxNonLODAvatars = 20,
```

Comment:
This should be in the graphics settings? This is related to the graphics rather than general information. Also, we should think a more friendly name so the users can understand what is for something like "Avatar rendering max distance" most users won't understand what LOD means

Reply:
Moved it to the graphics settings you are right.
We can change the naming in of the many PRs that will continue modifying this feature.